src/main/java/com/gregtechceu/gtceu/common/machine/electric/WorldAcceleratorMachine.java

```diff
@@ -0,0 +1,232 @@
+package com.gregtechceu.gtceu.common.machine.electric;
+
+import com.gregtechceu.gtceu.GTCEu;
+import com.gregtechceu.gtceu.api.GTValues;
+import com.gregtechceu.gtceu.api.blockentity.PipeBlockEntity;
+import com.gregtechceu.gtceu.api.machine.IMachineBlockEntity;
+import com.gregtechceu.gtceu.api.machine.TickableSubscription;
+import com.gregtechceu.gtceu.api.machine.TieredEnergyMachine;
+import com.gregtechceu.gtceu.api.machine.WorkableTieredMachine;
+import com.gregtechceu.gtceu.api.machine.trait.NotifiableEnergyContainer;
+import com.gregtechceu.gtceu.config.ConfigHolder;
+import com.gregtechceu.gtceu.utils.GTUtil;
+import com.lowdragmc.lowdraglib.syncdata.annotation.DescSynced;
+import com.lowdragmc.lowdraglib.syncdata.annotation.Persisted;
+import com.lowdragmc.lowdraglib.syncdata.field.ManagedFieldHolder;
+import it.unimi.dsi.fastutil.objects.Object2BooleanFunction;
+import it.unimi.dsi.fastutil.objects.Object2BooleanOpenHashMap;
+import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
+import lombok.Getter;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.entity.BlockEntityTicker;
+import net.minecraft.world.level.block.entity.BlockEntityType;
+import net.minecraft.world.phys.BlockHitResult;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static com.gregtechceu.gtceu.common.data.GTMachines.defaultTankSizeFunction;
+
+
+/**
+ * @author h3tr
+ * @date 2024/2/08
+ * @implNote WorldAcceleratorMachine
+ */
+
+public class WorldAcceleratorMachine extends WorkableTieredMachine  {
+
+    private static final Map<String, Class<?>> blacklistedClasses = new Object2ObjectOpenHashMap<>();
+    private static final Object2BooleanFunction<Class<? extends BlockEntity>> blacklistCache = new Object2BooleanOpenHashMap<>();
+
+    private static boolean gatheredClasses = false;
+
+    //Hard-coded blacklist for blockentities
+    private static final List<String> blockEntityClassNamesBlackList = new ArrayList<>();
+
+    protected static final ManagedFieldHolder MANAGED_FIELD_HOLDER = new ManagedFieldHolder(WorldAcceleratorMachine.class, TieredEnergyMachine.MANAGED_FIELD_HOLDER);
```

Comment:
should probably use `WorkableTieredMachine.MANAGED_FIELD_HOLDER` or na? OR just extend `TieredEnergyMachine`?
also fix the merge conflicts

Reply:
> should probably use `WorkableTieredMachine.MANAGED_FIELD_HOLDER` or na? OR just extend `TieredEnergyMachine`?

What do you mean with this? should I make the `WorldAcceleratorMachine` extend `TieredEnergyMachine`? Also I'm not very familiar with `ManagedFieldHolder`, what should I do with them?

Reply:
1. yes, because you aren't using any of the workable machine stuffs
2. it handles saving data with the annotations.